.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key:
+            ${{ matrix.os }}-${{ matrix.python-version }}-${{ hashFiles('**/requirements.txt') }}
```

Don't forget about the noxfile:
```suggestion
            ${{ matrix.os }}-${{ matrix.python-version }}-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('noxfile.py') }}
```

---

Does the noxfile define any requirements? Looks like it defers to requirements.txt?

```py
    session.install("-r", "requirements.txt")
```

Or do we want to clear the cache when the noxfile is modified anyway?

---

Not sure. My instinct is to include it in the cache key. There's also another session defined that we don't execute in CI, it installs `sphinx-autobuild` that is not present in normal requirements.

---

It's not a big deal either way, let's include it. Thanks!